Networking/Networking/Mapper/JWTokenMapper.swift

```diff
@@ -0,0 +1,90 @@
+
+import Foundation
+
+/// Mapper to parse the JWToken
+///
+struct JWTokenMapper: Mapper {
+    func map(response: Data) throws -> JWToken {
+        let decoder = JSONDecoder()
+        let jwt = try decoder.decode(JWTokenResponse.self, from: response).token
+
+        let payload: [String: Any]? = {
+            let parts = jwt.components(separatedBy: ".")
+            guard parts.count == 3 else {
+                return nil
+            }
+
+            let midPart = parts[1]
+
+            guard let bodyData = Data(base64URLEncoded: midPart) else {
+                return nil
+            }
+
+            guard let json = try? JSONSerialization.jsonObject(with: bodyData, options: []),
+                  let payload = json as? [String: Any] else {
+                return nil
+            }
+            return payload
```

Comment:
Nit: this can be simplified in one line

```suggestion
            return try? JSONSerialization.jsonObject(with: bodyData, options: []) as? [String: Any]
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/10896/commits/2b9268aa058057ebaa583c294e9f7df129b663e1